itou/www/login/forms.py

```diff
@@ -116,3 +117,23 @@ def login(self, request, redirect_url=None):
             accept_all_pending_invitations(request)
 
         return ret
+
+
+class ConfirmOTPForm(forms.Form):
+    otp_token = forms.CharField(required=True)
+
+    otp_token.widget.attrs.update(autocomplete="one-time-code")
```

Comment:
Est-ce qu’on ne voudrait pas faire ce code dans `__init__` ?
Comme le widget est cloné à l’`__init__()` du champ, ça marche. Mais les mutations à l’import me semblent plutôt dangereuses.

Reply:
C'était un suggestion de @leo-naeka

Reply:
C'est surtout que c'est une des méthodes privilégiées de Django : https://docs.djangoproject.com/en/5.1/ref/forms/widgets/#styling-widget-instances

Lorsque les champs sont définis sur le form et qu'il n'y a pas de raison de rendre la chose dynamique c'est  parfaitement indiqué, les attrs sont in fine [reconstruits dynamiquement](https://github.com/django/django/blob/5183f7c287a9a5d61ca1103b55166cda52d9c647/django/forms/widgets.py#L337-L339).

Reply:
Ohh, ok. Je ne pensais pas que c’était recommandé et j’avais un peu peur que ce soit abuser de la nature dynamique des widgets. Mais si c’est recommandé, mon erreur. AJA